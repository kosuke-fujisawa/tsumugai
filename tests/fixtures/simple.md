[SAY speaker=Hero]
Hello, world!

[WAIT 1s]

[SAY speaker=Hero]
Goodbye!