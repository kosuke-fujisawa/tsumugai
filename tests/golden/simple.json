{
  "input_hash": "4af8a800c01c7941496ceadc191ce2cf",
  "steps": [
    {
      "step_number": 0,
      "next_action": "WaitUser",
      "directives": [
        {
          "type": "Say",
          "args": {
            "speaker": "Hero",
            "text": "Hello, world!"
          }
        }
      ]
    },
    {
      "step_number": 1,
      "next_action": "WaitUser",
      "directives": [
        {
          "type": "Wait",
          "args": {
            "seconds": 1.0
          }
        }
      ]
    },
    {
      "step_number": 2,
      "next_action": "WaitUser",
      "directives": [
        {
          "type": "Say",
          "args": {
            "speaker": "Hero",
            "text": "Goodbye!"
          }
        }
      ]
    },
    {
      "step_number": 3,
      "next_action": "Halt",
      "directives": []
    }
  ],
  "final_state": "Completed"
}
